addons/website_slides/views/slide_slide_views.xml

```diff
@@ -149,10 +148,11 @@
             <field name="name">slide.slide.view.kanban</field>
             <field name="model">slide.slide</field>
             <field name="arch" type="xml">
-                <kanban edit="false">
+                <kanban edit="false" class="o_slide_channel_kanban">
```

Comment:
Why adding this class that is done for channel records ? Is it to have broader cards ? In which case we should change the naming to be more generic ...